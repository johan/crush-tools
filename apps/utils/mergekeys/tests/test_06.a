Month	Site	Ad Size	Ad Type	Country Category	Impressions
June	Page 3	468x60	Standard	US	1008405
June	Page 3	Not Selected	Standard	Others	160376
June	Page 3	Not Selected	Standard	US	252883
June	sun_SalesManager_test	160x600	Exclusive	UK	1927
June	sun_SalesManager_test	300x250	Exclusive	UK	2217
June	sun_SalesManager_test	728x90	Exclusive	UK	3229
June	TheLondonSite	125x125	Exclusive	EU	750
June	TheLondonSite	125x125	Exclusive	Others	1042
