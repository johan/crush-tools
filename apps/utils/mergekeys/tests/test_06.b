Month	Site	Ad Size	Ad Type	Country Category	Clicks
June	Page 3	468x60	Standard	US	9
June	Page 3	Not Selected	Time Based Bulk	EU	527
June	Page 3	Not Selected	Time Based Bulk	Others	1369
June	Page 3	Not Selected	Time Based Bulk	UK	2261
June	Page 3	Not Selected	Time Based Bulk	US	1481
June	The	125x125	Exclusive	EU	6
June	The	125x125	Exclusive	Others	25
