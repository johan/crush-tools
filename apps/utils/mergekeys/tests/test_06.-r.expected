Month	Site	Ad Size	Ad Type	Country Category	Impressions	Clicks
June	Page 3	468x60	Standard	US	1008405	9
June	Page 3	Not Selected	Time Based Bulk	EU		527
June	Page 3	Not Selected	Time Based Bulk	Others		1369
June	Page 3	Not Selected	Time Based Bulk	UK		2261
June	Page 3	Not Selected	Time Based Bulk	US		1481
June	TheLondonSite	125x125	Exclusive	EU	750	6
June	TheLondonSite	125x125	Exclusive	Others	1042	25
